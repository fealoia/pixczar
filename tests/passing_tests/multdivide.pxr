Void main() {
	Int x = 6;
	Int y = 3;
	print(x*y);
	print(x/y);

	Float a = 2.4;
	Float b = 1.2;
	print(a*b);
	print(a/b);
}
