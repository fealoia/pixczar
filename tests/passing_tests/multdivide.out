18
2
2.88
2
